examples/curations.yml

```diff
@@ -59,3 +59,9 @@
     comment: "The package has an unmappable declared license entry."
     declared_license_mapping:
       "BSD-derived (http://www.repoze.org/LICENSE.txt)": "LicenseRef-scancode-repoze"
+
+- id: "PyPI::branca"
+  curations:
+    comment: "This is a copyright statement, not a license."
```

Comment:
"This" sounds a bit as if it referred to itself as a sentence, i.e. to the comment. How about "A copyright statement was used to declare the license."?

Reply:
Right, Done!